MessageViewController/MessageAutocompleteController.swift

```diff
@@ -29,6 +29,27 @@ public final class MessageAutocompleteController: MessageTextViewListener {
         public let range: NSRange
     }
     public private(set) var selection: Selection?
+    
+    /// Adds an additional space after the autocompleted text when true. Default value is `TRUE`
+    open var appendSpaceOnCompletion = true
+    
+    /// The default text attributes
+    open var defaultTextAttributes: [NSAttributedStringKey: Any] =
+        [NSAttributedStringKey.font: UIFont.preferredFont(forTextStyle: .body),
+         NSAttributedStringKey.foregroundColor: UIColor.black]
+    
+    /// The text attributes applied to highlighted substrings for each prefix
+    open var autocompleteTextAttributes: [String: [NSAttributedStringKey: Any]] = [:]
+    
+    /// A key used for referencing which substrings were autocompletes
+    private let NSAttributedAutocompleteKey = NSAttributedStringKey.init("com.system.autocompletekey")
```

Comment:
nit: Could we namespace w/ the project name? So
```swift
private let NSAttributedAutocompleteKey = 
  NSAttributedStringKey.init("com.messageviewcontroller.autocompletekey")
```

Reply:
👍